internal/show/connections.go

```diff
@@ -0,0 +1,140 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package show
+
```

Comment:
I see that you moved _brokers.go_ to preserve the git history but copied this and the others. I know we have to temporarily leave some remnants behind under _pkg/subctl/cmd/show_ but we could still move these files and put what's left, ie creation of the `cobra.Command`, in _pkg/subctl/cmd/show/show.go_, same as you did in _cmd/subctl/show.go_.

Reply:
I have moved all logic to `/internal/show` and just kept cobra command in `pkg/subctl/cmd/show/*.go`. I haven't made any changes there because it is eventually being deleted.

Reply:
My point is if we move _pkg/subctl/cmd/show/*.go_  instead of copying we can keep the git history. The cobra-related remnants in the original files can simply be put in _show.go_.